src/view/Pages/Patients/edit/EditPatient.js

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from "react";
 import Button from "../../../components/Button/Button";
 import "./EditPatient.scss";
+import { useDispatch } from "react-redux";
+import { updatePatientsRequest } from "../../../store/patientActions";
 
 const EditPatient = ({ patient, onCloseEditModal }) => {
   const [editedPatient, setEditedPatient] = useState({
+    hospitalId: patient.hospitalId,
     fullName: patient.fullName,
     age: patient.age,
-    gender: patient.gender,
```

Comment:
Why gender is still showing as removed param?

Reply:
for gender field will handle that on pms-38 which is create and edit modal  after we get pull of pms-47. once gender wil add then update request will work.